Content/LeagueSandbox-Scripts/LeagueSandbox-Scripts.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
-    <Platforms>AnyCPU;x64</Platforms>
+    <Platforms>AnyCPU;x86</Platforms>
```

Comment:
❓

Reply:
x86 was the only platform GameServer would build with on Windows as far as I tested.

Reply:
This only makes me wonder what was preventing GameServerConsole from building on other architectures, there shouldn't be anything that makes it required

Reply:
I believe it was a DLL issue with ENet, however even when I tested x32 and x64 built DLLs, it did not resolve the issue.

Reply:
Building on AnyCPU on Windows should give you the exact error.

Reply:
GameServerConsole shouldn't have a dependency to ENet, right?

Reply:
Interestingly enough I was sure x64 was the only build shipped with nuget, I might be wrong though. The repo for that is here: https://github.com/LeagueSandbox/ENetSharpLeague and build config here: https://github.com/LeagueSandbox/ENetSharpLeague/blob/indev/ENetCS/ENetCS.csproj

Reply:
Honestly no idea, It runs fine with x86, but not x64, x32, or Any CPU. The problem is with GameServerConsole in particular. Setting its platform target to x86 for Any CPU also fixes it, but I didn't want to do this as it seems to break the build for Linux users.